R/calc_addressbase_plus_single_line_address.R

```diff
@@ -106,129 +68,50 @@ calc_addressbase_plus_geo_single_line_address <- function(
-            is.null(SAO_END_NUMBER),
-          yes = paste0(SAO_START_NUMBER, ", "),
-          no = ifelse(
-            test = is.null(SAO_START_NUMBER),
-            yes = "",
-            no = as.character(SAO_START_NUMBER)
-          )
-        ),
-        ifelse(
-          test = !is.null(SAO_START_SUFFIX) & is.null(SAO_END_NUMBER),
-          yes = paste0(SAO_START_SUFFIX, ", "),
-          no = ifelse(
-            test = !is.null(SAO_START_SUFFIX) & !is.null(SAO_END_NUMBER),
-            yes = SAO_START_SUFFIX,
-            no = ""
-          )
-        ),
-        ifelse(
-          test = !is.null(SAO_END_SUFFIX) & !is.null(SAO_END_NUMBER),
-          yes = "-",
-          no = ifelse(
-            test = !is.null(SAO_START_NUMBER) & !is.null(SAO_END_NUMBER),
-            yes = "-",
-            no = ""
-          )
-        ),
-        ifelse(
-          test = !is.null(SAO_END_NUMBER) & is.null(SAO_END_SUFFIX),
-          yes = paste0(SAO_END_NUMBER, ", "),
-          no = ifelse(
-            test = is.null(SAO_END_NUMBER),
-            yes = "",
-            no = as.character(SAO_END_NUMBER)
-          )
-        ),
-        ifelse(
-          test = !is.null(PAO_TEXT),
-          yes = paste0(PAO_TEXT, ", "),
-          no = ""
-        ),
-        ifelse(
-          test = !is.null(PAO_START_NUMBER) &
-            is.null(PAO_START_SUFFIX) &
-            is.null(PAO_END_NUMBER),
-          yes = paste0(PAO_START_NUMBER, ", "),
-          no = ifelse(
-            test = is.null(PAO_START_NUMBER),
-            yes = "",
-            no = as.character(PAO_START_NUMBER)
-          )
-        ),
-        ifelse(
-          test = !is.null(PAO_START_SUFFIX) & is.null(PAO_END_NUMBER),
-          yes = paste0(PAO_START_SUFFIX, ", "),
-          no = ifelse(
-            test = !is.null(PAO_START_SUFFIX) & !is.null(PAO_END_NUMBER),
-            yes = PAO_START_SUFFIX,
-            no = ""
-          )
-        ),
-        ifelse(
-          test = !is.null(PAO_END_SUFFIX) & !is.null(PAO_END_NUMBER),
-          yes = "-",
-          no = ifelse(
-            test = !is.null(PAO_START_NUMBER) & !is.null(PAO_END_NUMBER),
-            yes = "-",
-            no = ""
-          )
-        ),
-        ifelse(
-          test = !is.null(PAO_END_NUMBER) & is.null(PAO_END_SUFFIX),
-          yes = paste0(PAO_END_NUMBER, ", "),
-          no = ifelse(
-            test = is.null(PAO_END_NUMBER),
-            yes = "",
-            no = as.character(PAO_END_NUMBER)
-          )
-        ),
-        ifelse(
-          test = !is.null(STREET_DESCRIPTION),
-          yes = paste0(STREET_DESCRIPTION, ", "),
-          no = ""
-        ),
-        ifelse(
-          test = !is.null(LOCALITY),
-          yes = paste0(LOCALITY, ", "),
-          no = ""
-        ),
-        ifelse(
-          test = !is.null(TOWN_NAME),
-          yes = paste0(TOWN_NAME, ", "),
-          no = ""
-        )
+      SAO_DASH = ifelse(
```

Comment:
To be able to add in dashes as appropriate, need to do this before using `unite.tbl`